File: wallendar-app/app/src/main/java/com/td/wallendar/groupmembers/GroupMembersViewHolder.java
package com.td.wallendar.groupmembers;

import android.view.View;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.td.wallendar.R;
import com.td.wallendar.models.ApplicationUser;

public class GroupMembersViewHolder extends RecyclerView.ViewHolder {

    public GroupMembersViewHolder(@NonNull View itemView) {
        super(itemView);
    }

    public void bind(final ApplicationUser applicationUser) {
        final TextView textView = itemView.findViewById(R.id.member_text_name);
        String userNameText = applicationUser.getFirstName() + " " + applicationUser.getLastName();
        textView.setText(userNameText);

    }

}

File: wallendar-app/app/src/main/java/com/td/wallendar/groupmembers/GroupMembersViewHolder.kt
package com.td.wallendar.groupmembers

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.td.wallendar.R
import com.td.wallendar.models.ApplicationUser

class GroupMembersViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    fun bind(applicationUser: ApplicationUser) {
        val textView = itemView.findViewById<TextView?>(R.id.member_text_name)
        val userNameText = applicationUser.firstName + " " + applicationUser.lastName
        textView.text = userNameText
    }
}